### PR TITLE
Demolish skyscrapers

### PR DESCRIPTION
**Pull request by cliffckerr**
### Brief description
Minor change to `calibrate_manual.py` to change `bar3d` to `pcolormesh`, and adds the test script. The skyscrapers were already converted to heatmaps in the library code (`experiment.py`) in May 2022.

[NB: test are failing, but they are on `main` as well.]

### Type(s) of change
- [ ] Bugfix
- [x] Refactor
- [ ] New feature
- [ ] Other

### Checklist
- My code follows the [style guide](https://github.com/amath-idm/styleguide)
  - [x] Yes
  - [ ] N/A
- I've commented my code
  - [x] Yes
  - [ ] N/A
- I've incremented the version number
  - [ ] Yes
  - [x] N/A
- I've updated the changelog
  - [ ] Yes
  - [x] N/A
- I've added tests and checked code coverage
  - [ ] Yes
  - [x] N/A

### CODE REVIEW

**Comment by emilydriano** [comment]
I think if we just merge main into this branch, looks good to go!

**Comment by MObrien-IDM** [comment]
LGTM @cliffckerr - once you merge in main so that the tests pass, feel free to ping me for sign off :)

**Review by MObrien-IDM — APPROVED** [review]

**Pull request merged**
